comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -284,6 +281,13 @@ func (ia *inventoryagent) fetchSecurityAgentMetadata() {
 	ia.data["feature_cspm_host_benchmarks_enabled"] = securityCfg.GetBool("compliance_config.host_benchmarks.enabled")
 }
 
+func (ia *inventoryagent) fetchTraceAgentMetadata() {
+	traceCfg, _ := ia.getCorrectConfig("trace-agent", fetchTraceConfig)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏What's happening when the fetch fails? I would like to avoid issues where the metadata flaps between payloads.

Reply:
We fallback on the core-agent config. Since the paylaod is sent every 10 min I don't think it's an issue.